package.json

```
@@ -12,6 +12,7 @@
     "@wireapp/protocol-messaging": "1.23.0",
     "@wireapp/react-ui-kit": "5.1.4",
     "amplify": "git+https://git@github.com/wireapp/amplify",
+    "core-js": "2",
```

Let's specify a complete version here
```suggestion
    "core-js": "2.6.5",
```

---

done